### ISSUE TITLE
[FEATURE REQUEST] - drag to other app, simple mode by default

### ISSUE THREAD

**Issue by thewayur** [receyuki/stable-diffusion-prompt-reader#47]
### Summary

1. Drag to other to other apps
2.  simple mode.

i have other requests too such as 
a) 100% full screen image preview. (removing all the extra top, bottom borders.

b) as u can see in screenshot, the data is not displayed well (i m using very latest release), please let us view raw(json/xml or waterver) data too.

### Basic Example

hi, i use this app on regular bases. So according to my usage i would like to suggest features.

1. Drag to other to other apps

> so just like breadboard, there should be drag functionality. so that when we drag the pic to comfiui/imgtoimg or file explorer, it sends the image to that app.

2.  simple mode.

> prompts in single line is ok, but when it comes to other info like MODEL, CFG SCALE, it will be more convinient for us to view that info in simple mode by default. please look below screenshots for  references

![image](https://github.com/receyuki/stable-diffusion-prompt-reader/assets/2944767/67cde126-30c2-4c0d-a55d-a2d6dce7fd72)

SIMPLE MODE
![image](https://github.com/receyuki/stable-diffusion-prompt-reader/assets/2944767/99459c00-ec76-4e0a-946d-e3ce42f2b794)

a) i have other requests too such as 100% full screen image preview. (removing all the extra top, bottom borders.
b) as u can see in screenshot, the data is not displayed well (i m using very latest release), please let us view raw(json/xml or waterver) data too.

thanks a lot for this app, i simply love it

**Comment by receyuki** [comment]
Thank you so much for your feedback.

- Based on my attempts, it seems that the GUI framework I'm using doesn't support dragging files out of the window (even ChatGPT confirmed it's not possible). However, you can give my recently released [ComfyUI node version of SD Prompt Reader](https://github.com/receyuki/comfyui-prompt-reader-node) a try; it might be helpful to you.
- I currently plan to add a user preference panel in either version 1.4.0 or 1.4.1, where I will include an option to choose the default mode.
- Version 1.4.0 will bring significant changes to the GUI. I am considering adding a full-screen button that opens image in a new window. When you switch images, the image in the new window will also change accordingly. Of course, you can close this window at any time.
- When you use custom nodes or the workflow becomes too complex, there is a possibility that metadata cannot be read. This is because ComfyUI does not store metadata but only the complete workflow. Therefore, even if I display the original JSON, I believe it would be challenging for users to manually extract the data they need from thousands or even tens of thousands of lines of raw data. I have manually adapted each custom node mentioned in the issues (which is a significant workload and is a task that never ends). But now, I hope that the [ComfyUI node version of SD Prompt Reader](https://github.com/receyuki/comfyui-prompt-reader-node) can change all of this.

**Comment by thewayur** [comment]
i m pretty excited for upcoming versions. Thanks for making our life simpler.

for the "raw json/data" display thng. i would suggest u to leave that up to user.
i can u suggest u something like this(if u  have extra time to do it) =
1. a tab/area for json/raw data view. (doesnt need to be formatted at  all)
2. that text area would be editable for power users like me.
3.  a button to search a query / highglight a word. which lead us to a particular line. (just like  f3 in chorme)

this will help us search queries like = safetensor
& boom. we can navigate through all safetensors words. Profit

i would like to request new feature time to time.(sorry, i am pretty greedy :D )

*thanks again to hear us*

**Comment by receyuki** [comment]
@thewayur 
I'm not sure if you are aware, but the "Copy" button to the right of "Export" can directly copy the original JSON metadata of the image. For example, for a comfy image, clicking the "Copy" button will copy the content as 
```
positive prompt
negative prompt 
original JSON metadata (Including the complete workflow and all node data)
``` 
Personally, I prefer using external editors to view it, such as VSCode. After all, these well-known editors are much more user-friendly than anything I could write myself.
So, I'm considering whether adding an option to open the metadata with an external editor on the "Copy" button might be a better choice.

**Comment by thewayur** [comment]
@receyuki yes, i use vscode most of times. and i m using same in this situation. well its fine. 
thanks  for consideration  and clearing things up  for my mind state :)
 
u can close this topic :)
